css/prosemirror.scss

```diff
@@ -2,7 +2,7 @@
 
 /* Document rendering styles */
 div.ProseMirror {
-	margin-top: 44px;
+	// margin-top: 44px;
```

Comment:
Looks unrelated

Reply:
@vinicius73 Can you check that, then we could get this in so we can get the PR list down a bit :)

Reply:
Yes, it's not related, but I need to send another PR to fix it.